pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,78 @@
+{ stdenv, fetchFromGitHub, fetchurl, lib, platforms,
+  mariadb, mariadb-connector-c,
+  boost, cmake, bison, flex, }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, fetchurl, lib, platforms
, mariadb, mariadb-connector-c
, boost, cmake, bison, flex
}:
```

Reply:
Thanks, updated. Is there a linter or formatting documentation I should use for these sort of things?

Reply:
The contributing guide has a section about formatting.